### DISCUSSION TITLE
[Docs addon] How can I make "Docs" item the last of each list?

### DISCUSSION THREAD

**Discussion by sneko** [storybookjs/storybook#20684] [Help]
Hi,

Currently by default I have:
<img width="187" alt="image" src="https://user-images.githubusercontent.com/8195958/213410634-eed359bb-bf8b-43c9-ae0d-54e938ecf0d9.png">

But I would like to have instead:
- `Normal`
- `Docs`

Since I'm on Storybook v7 the `storySort` does not work, it seems there are possibilities to enable legacy feature to make it working but I would like to avoid "workarounds".

Do you know a way to just tell the Docs addon to display as the last item each time?

Thank you,

EDIT: other acceptable solution would be when clicking `AgentCard` it opens always the story just after `Docs`.

**Comment by paulwoodland** [comment]
We would also like to able to do this, as the stories themselves are usually what people are after rather than the docs page. I haven't found any way of doing this so far.

**Comment by zecka** [comment]
@sneko did you find a solution ?

In my side the only solution i found is like bellow. But this is not allow you to use a complexe array order like you can have with the basic storySort configuration


```ts
  
storySort: (a, b) => {
  // Order for the root categories
  const order = [
    'Atoms',
    'Molecules',
    'Organisms',
  ];
  const aPaths = a.title.split('/');
  const bPaths = a.title.split('/');
  const rootA = aPaths[0];
  const rootB = bPaths[0];

  const indexA = order.indexOf(rootA);
  const indexB = order.indexOf(rootB);

  if (indexA === indexB) {
    if (a.type === 'docs') {
      return 1;
    } else if (b.type === 'docs') {
      return -1;
    }
    return 0;
  }
  return indexA > indexB ? 1 : -1;
},
```

**Reply by sneko** [reply]
No solution found sorry 😢
